test/html/test.ml

```diff
@@ -107,16 +107,49 @@ let () =
     let reference_file = expect_directory // lang // (file_title ^ ".html") in
 
     let validate_html file =
+      let label = "tidy" in
+      let muted_warnings = String.concat "," [
```

Comment:
These are the 3 warnings that we said we didn't want to fail on — however, `MISSING_STARTTAG` is going to pop up for _a lot_ of tags. Some of which may or may not be optional by https://html.spec.whatwg.org/ but that `tidy` will consider not optional.

Exhaustively listing all of these can be done by parsing the HTML Spec, but I'd refrain from doing all that work unless we absolutely need to.

Reply:
What is triggering `MISSING_STARTTAG` in the odoc output? Both `MISSING_STARTTAG` and `DISCARDING_UNEXPECTED` look uncomfortably broad to me (we may have to accept that, though :/)

Reply:
The missing `<dd>` tags on the `<dl>`'s are triggering `MISSING_STARTTAG` and some `<dt>`s are triggering `DISCARDING_UNEXPECTED`.

I haven't managed to find what exactly was, within the `<dt>`, that triggered the error, but my thesis is that `tidy` is behind what the HTML Spec considers [Flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2) (which is what's allowed within `<dt>`).

Reply:
If there's no way to silence the warning only for `<dd>`, we can leave it as in this PR now. Presumably, we still get a lot of value from tidy's other warnings.